css/style.css

```diff
@@ -2,19 +2,15 @@ body, html {
     margin: 0;
     padding: 0;
 }
-.container{
-    width: 80%;
-    margin-left: 10%;
+.lb-container{
+    padding: 15px;
+    background-color: rgba(255, 255, 255, .2);
 }
-#list li{
-    list-style: none;
-    margin: 0 auto;
+#images img{
+    margin-top: 15px;
 }
-#list img{
-    width: 100px;
-    position: relative;
-    display: inline-block;
-}
-.imgArray{
+.col-xs-6, .col-md-3, .col-lg-1{
+    float: none;
     display: inline-block;
+    vertical-align: middle;
```

Comment:
@cxoltero 
This is what I was trying to avoid. Here, you're overriding the bootstrap classes, when you should create a custom class (something like "column-float-fix") and then add that class to the HTML. By overriding the default bootstrap classes, you potentially break other parts of the layout. In this app, there isn't much more layout, but in a larger app, these column classes could appear dozens of times on a single page.